frontend/src/plugin/registry.tsx

```diff
@@ -297,6 +302,10 @@ export function registerSidebarEntry({
   );
 }
 
+export function registerKubeObjectGlance(glance: Glance) {
```

Comment:
These kind of functions are used by plugin authors and so require a thorough documentation. Please add a description with an example. Check out how other reigster functions are documented

Reply:
In addition to this, we also have a website with the documentation, a markdown file `extending-the-map.md` contains explanation on how to extend the map and this function should be added there.
https://headlamp.dev/docs/latest/development/plugins/functionality/extending-the-map
https://github.com/kubernetes-sigs/headlamp/blob/main/docs/development/plugins/functionality/extending-the-map.md